### PR TITLE
Feat/admob adapters zones support and global ad view and fix skipped ads

### PR DESCRIPTION
**Pull request by christophercong**

### CODE REVIEW

File: AdMob/iOS/GADMAdapterAppLovinRewardBasedVideoAd.m [hunk 1]
```diff
@@ -70,6 +73,21 @@ - (void)requestRewardBasedVideoAd
     NSString *adapterVersion = [[self class] adapterVersion];
     [[ALSdk shared] setPluginVersion: adapterVersion];
     
+    // Zones support is available on AppLovin SDK 4.5.0 and higher
+    NSString *zoneIdentifier = [self zoneIdentifierFromConnector: self.connector];;
```
Comment (thomasmso) [inline AdMob/iOS/GADMAdapterAppLovinRewardBasedVideoAd.m]: Can you remove the extra `;`

File: AdMob/iOS/GADMAdapterAppLovinRewardBasedVideoAd.m [hunk 2]
```diff
@@ -70,6 +73,21 @@ - (void)requestRewardBasedVideoAd
...
+    }
+    else
+    {
+        self.incent = [[ALIncentivizedInterstitialAd alloc] initWithSdk: [ALSdk shared]];
+    }
+    
+    self.incent.adVideoPlaybackDelegate = self;
+    self.incent.adDisplayDelegate = self;
+    
     if ( self.incent.readyForDisplay )
```
Comment (thomasmso) [inline AdMob/iOS/GADMAdapterAppLovinRewardBasedVideoAd.m]: I thought you made a fix for `self.incent.readyForDisplay` always evaluating to `NO`?

**Comment by christophercong** [comment]
Changes made.

**Comment by christophercong** [comment]
@thomasmso  Changes made to support zone_id thru the AdMob server parameter.

**Pull request merged**